maro/rl/rollout/env_sampler.py

```diff
@@ -192,9 +192,11 @@ class AbsEnvSampler(object, metaclass=ABCMeta):
 
     Args:
         get_env (Callable[[], Env]): Function used to create the rollout environment.
-        policy_creator (Dict[str, Callable[[str], AbsPolicy]]): Dict of functions that used to get policies, specified
-            by policy names.
-        agent2policy (Dict[Any, str]): Mapping of agent name and policy name.
+        policy_creator (Dict[str, Callable[[str], AbsPolicy]]): Dict of functions to create policies by name.
+        agent2policy (Dict[Any, str]): Mapping of agent name to policy name.
+        trainable_policies (List[str], default=None): List of trainable policy names. Experiences generated using the
```

Comment:
Can we add a property to `AbsPolicy`, and let the property indicate whether it is trainable or not?

Reply:
It is not a property. Even an RLPolicy can be fixed (e.g., when using some pre-trained RL policy). We let the users define which policies they want to train.